addons/crm_iap_mine/views/crm_iap_lead_mining_request_views.xml

```diff
@@ -68,13 +63,13 @@
                             <field name="industry_ids" widget="many2many_tags" attrs="{'readonly': [('state', '=', 'done')]}" options="{'no_create': True, 'no_open': True}" class="o_industry" required="1"/>
                             <field name="filter_on_size" attrs="{'readonly': [('state', '=', 'done')]}"/>
                             <label for="company_size_min" attrs="{'invisible': [('filter_on_size', '=', False)]}"/>
-                            <div attrs="{'invisible': [('filter_on_size', '=', False)]}">
+                            <span attrs="{'invisible': [('filter_on_size', '=', False)]}">
```

Comment:
Isn't it possible to keep a div ? That way we lessen potential issues with existing DB in 16 ...